source/Willow-Bootstrap-Tests/BootstrapSelectTest.class.st

```diff
@@ -135,7 +135,7 @@ BootstrapSelectTest >> testDropdownAlignRight [
 
 	| html |
 
-	html := self renderMultiSelectApplying: [ :bootstrapSelect | bootstrapSelect dropdownAlignRight: BootstrapSelectConstants auto ].
+	html := self renderMultiSelectApplying: [ :bootstrapSelect | bootstrapSelect dropdownAlignRight: bootstrapSelect constants >> #bootstrap >> #auto ].
```

Comment:
This should not need to be like this.  Something like `BootstrapSelect constants >> #auto` should be enough. It can't be right to write bootstrap twice in the same collaboration.